requirements/script/lemonldap-entrypoint.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+service cron start
+service anacron start
+
+
```

Comment:
for PRESERVEFILE in ${PRESERVEFILES} ;
do
    if [ ! "$(ls -A ${PRESERVEFILE} &>/dev/null)" ]; then
        echo "# Restore ${PRESERVEFILE} directory"
        chown -R www-data:www-data ${PRESERVEFILE}
        cp -a ${PRESERVEFILE}-orig/* ${PRESERVEFILE}/
    fi
done